.circleci/config.yml

```diff
@@ -361,17 +361,22 @@ jobs:
     executor: node-browsers
     steps:
       - run: *shallow-git-clone
+      - run:
+          name: Save Yarn version
+          command: |
+            yarn --version > /tmp/YARN_VERSION
```

Comment:
A file was used here rather than an environment variable because CircleCI doesn't support referencing dynamically-set environment variables from restore_cache keys. It only works for environment variables set statically for the entire workflow.